src/OrchardCore.Themes/TheAdmin/Assets/scss/components/_validations.scss

```diff
@@ -112,3 +112,14 @@ textarea.form-control.input-validation-error {
     border-color: var(--bs-form-invalid-border-color);
     padding: 0.375rem 0.75rem;
 }
+.has-validation-error .multiselect__tags {
+    border: 1px solid red !important;
+    background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 12 12' width='12' height='12' fill='none' stroke='%23dc3545'%3e%3ccircle cx='6' cy='6' r='4.5'/%3e%3cpath stroke-linejoin='round' d='M5.8 3.6h.4L6 6.5z'/%3e%3ccircle cx='6' cy='8.2' r='.6' fill='%23dc3545' stroke='none'/%3e%3c/svg%3e");
+    background-repeat: no-repeat;
+    background-position: right calc(0.375em + 0.1875rem) center;
+    background-size: calc(0.75em + 0.375rem) calc(0.75em + 0.375rem)
+}
+
+.has-validation-error .multiselect__select {
+    padding-right: calc(1.5em + 0.95rem)
+}
```

Comment:
Don't repeat styling. Try to reuse the `input-validation-error` class on an element. Better change the HTML structure of the editor to make it compatible with it than add a custom validation error CSS just for this case.

And as mentioned previously, if new CSS is needed, you'll need to run SCSS compilation with Gulp: https://docs.orchardcore.net/en/latest/guides/gulp-pipeline/. If something is Content Picker-specific, then it also needs to be added only the `OrchardCore.ContentFields` module, and included from only this cshtml. If you search for "optionsEditor.css" in that project, you'll see an end-to-end example of this this can be done.